src/style.css

```diff
@@ -0,0 +1,99 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+.page {
+  --big-bar-height: 70px;
+  --big-bar-pleft: 62px;
+  --big-bar-border: 1px;
+  --big-search-icon-top: calc(var(--big-bar-height) / 2);
+
+  --small-bar-height: 42px;
+  --small-bar-pleft: 33px;
+  --small-bar-border: 1px;
+  --small-search-icon-top: calc(var(--small-bar-height) / 2);
+
+  font-family: "Avenir-Book", sans-serif;
+  font-weight: 300;
+}
+/*
+.page-body {
+  margin: 0;
+  padding: 0;
+} */
+
+.container {
+  margin-top: 20px;
+  position: relative;
+  display: block;
+  height: var(--big-bar-height);
+}
+
+.container--small {
+  height: var(--small-bar-height);
```

Comment:
i dont think you need to hardcode the height. i explained in one of the comments